Examples/Swift/UserTrackingModesExample.swift

```diff
@@ -9,23 +9,24 @@ class UserTrackingModesExample_Swift: UIViewController, MGLMapViewDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        // Create a new Mapbox map using the Mapbox Dark style
```

Comment:
Should we explain this boilerplate? For non-style-centric examples, I think we’ve tended to let this go without comment.